### PR TITLE
feat: add version history

### PR DESCRIPTION
**Pull request by Benjozork**
Fixes part of #25

## Summary of Changes

* Adds "Version history" section to aircraft page

## Screenshots (if necessary)

image

## Additional context

Figma

image

Discord username (if different from GitHub):

### CODE REVIEW

File: src/renderer/components/App/GitHubApi.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        return ghReleases.map(release => {
+            const dateDay = release.created_at.substring(8, 10);
+            const dateMonth = release.created_at.substring(5, 7);
+            const dateYear = release.created_at.substring(2, 4);
+
+            return {
+                title: release.tag_name,
+                date: `${dateDay}/${dateMonth}/${dateYear}`,
+                type: 'minor'
```
Comment (ZigTag) [inline src/renderer/components/App/GitHubApi.ts:25]: could probably implement a detection system if PATCH >= 1 or MINOR >= 1

**Review by ZigTag — CHANGES_REQUESTED** [review]

File: src/renderer/components/App/GitHubApi.ts (hunk 1 above)
Comment (Benjozork) [reply src/renderer/components/App/GitHubApi.ts:30]: Done

**Review by ZigTag — CHANGES_REQUESTED** [review]

File: src/renderer/components/AircraftSection/VersionHistory.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ * Container for versions
+ */
+export const Versions = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const GITHUB_RELEASE_BASE_URL = 'https://github.com/flybywiresim/a32nx/releases/tag/';
+
+type VersionProps = { index: number, className?: string, version: ModVersion };
```
Comment (ZigTag) [inline src/renderer/components/AircraftSection/VersionHistory.tsx:16]: Add newlines

Comment (Benjozork) [reply src/renderer/components/AircraftSection/VersionHistory.tsx:16]: bruh

**Comment by ilyeshammadi** [comment]
I'm a bit late to the review but here is mine:
- VersionHistroy.tsx should be `components/VersionHistroy` with `index.tsx` and `styles.tsx` 
- The component is missing a loading state while the data is being fetched.
- Font-size of the version is a bit big in my opinion

File: src/renderer/components/AircraftSection/VersionHistory.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            <VersionDate>{props.version.date}</VersionDate>
+            <VersionType>{props.version.type} version</VersionType>
+        </div>
+    );
+};
+
+/**
+ * Visually displays a mod release
+ */
+export const Version = styled(VersionBase)`
```
Comment (ilyeshammadi) [inline src/renderer/components/AircraftSection/VersionHistory.tsx:53]: **nitpick:** styles should be in `VersionHistroy/styles.tsx`

File: src/renderer/components/AircraftSection/VersionHistory.tsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  display: grid;
+  grid-template-columns: [start] 1fr [middle] 6fr [end] 3fr;
+  grid-template-rows: [start] 30px [middle] auto;
+
+  letter-spacing: .04em !important;
+  
+  --timeline-node-color: ${props => props.index === 0 ? 'white' : '#757575'};
```
Comment (ilyeshammadi) [inline src/renderer/components/AircraftSection/VersionHistory.tsx:60]: **suggestion:** Ant design has already a timeline component if you are not aware of that
https://ant.design/components/timeline/

File: src/renderer/components/AircraftSection/VersionHistory.tsx [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+/**
+ * Container for versions
+ */
+export const Versions = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const GITHUB_RELEASE_BASE_URL = 'https://github.com/flybywiresim/a32nx/releases/tag/';
```
Comment (ilyeshammadi) [inline src/renderer/components/AircraftSection/VersionHistory.tsx:14]: **suggestion:** It's better to keep constants in a separate file at the root level.

**Comment by St54Kevin** [comment]
Maybe make the current build number a different color like orange, when the installed build ist not the actual current one? And make it green when installed?

**Pull request merged**